docs/notebooks/downstream_machine_learning_classification_evaluation.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluate synthetic data on downstream classification models\n",
+    "\n",
+    "### How to use this notebook\n",
+    "Use this notebook to understand the quality and performance of your synthetic or augmented data on downstream machine learning classification tasks. \n",
```

Comment:
suggesiton: Can we be more explicit that we're comparing synthetic to real data performance here? Real data isn't mentioned at all right now.

And did we decide on terminology for "synthetic" or "synthetic or augmented", I think I saw a discussion but didn't catch the resolution.